packages/create-turbo/src/utils/git.ts

```diff
@@ -0,0 +1,88 @@
+import fs from "fs-extra";
+import { execSync } from "child_process";
+import path from "path";
+import rimraf from "rimraf";
+
+export const DEFAULT_IGNORE = `
```

Comment:
It would be nice if this was somewhere else, idk how we'd bundle it though... just handy to have for others. There is a repo that hosts .gitignore files maybe we can consider opening a pr there and just downloading it (or a comment with a link to it) so we have a canonical version.

Reply:
Yes, what we really care about here is .turbo, but this is a good call, maybe we can augment another shared one.

Reply:
I think I'm going to punt this one for now, and possible come back to it when we have support for dynamic transforms that could take care of this in a more repeatable way.